### DISCUSSION TITLE
problème déploiement sur plusieurs sites

### DISCUSSION THREAD

**Discussion by HoboWanderer** [AmauriC/tarteaucitron.js#712] [Q&A] [answered]
Bonjour,

J'ai un problème. Je dois déployer Tarteaucitron sur un peu plus de 400 sites qui ont un nom de domaine commun.

Exemple : 
Site 1 : ht.tps://monnomdedomaine.fr/site1/
Site 2 ht.tps://monnomdedomaine.fr/site2/

Une fois TAC déployé sur chaque site, je me rend sur site1, je règle mes autorisations. Je souhaite me rendre ensuite sur site2 pour autoriser les règles différemment mais ils se trouvent qu'elles sont déjà validés avec la configuration de site1....

Comment faire pour bien départager chaque site ? 
Je ne souhaite pas que TAC se base sur le mon nom de domaine mais sur le premier répertoire après le nom de domaine... Est-ce possible ?

Merci déjà pour ce service
Cordialement,

**Comment by HoboWanderer** [comment, marked as answer]
Bon, je ne trouve pas le bouton pour supprimer cette demande... Je n'avais tout simplement pas réfléchi à changer le nom de cookie par défaut.... Mea Culpa ;)

**Reply by AmauriC** [reply]
C'est la solution que j'allais proposer :)

La discussion pourra peut-être servir à quelqu'un d'autre ; )
